src/clj/fluree/db/api/query.cljc

```diff
@@ -160,11 +171,16 @@
   (go-try
     (try*
       (let [[alias explicit-t] (extract-query-string-t alias)
-            address (<? (nameservice/primary-address conn alias nil))
-            ledger  (<? (jld-ledger/load conn address))
-            db      (ledger/-db ledger)
-            t*      (or explicit-t t)]
-        (<? (restrict-db db t* context opts)))
+            address            (<? (nameservice/primary-address conn alias nil))
+            ledger             (<? (jld-ledger/load conn address))
+            db                 (ledger/-db ledger)
+            t*                 (or explicit-t t)
+            rules-db           (let [dbs-or-aliases (:reasoner-rules-db opts)]
+                                 (if (string? (first dbs-or-aliases))
+                                   [(ledger/-db (<? (jld-ledger/load conn (first dbs-or-aliases))))]
+                                   dbs-or-aliases))
+            opts*              (assoc opts :reasoner-rules-db rules-db)]
```

Comment:
Very minor, but if possible can you configure your editor to ignore destructuring forms for alignment purposes? They just tend to get pretty long and make the other bindings in the let form align too far to the right. So many exceptions to every rule... ;)